backscatter/fitacf/fitacf.py

```diff
@@ -459,43 +461,50 @@ def _fit(raw_data, debug_mode=False):
     if debug_mode:
         debug_output(range_list, raw_data)
 
-    determined_parameters = Determinations(raw_data,range_list,noise_pwr)
+    determined_parameters = Determinations(raw_data,range_list,noise_pwr, tdiff)
     return determined_parameters.paramater_dict
 
 
-def fit(raw_records):
+def fit(raw_records, tdiff: float = None):
     """Performs the whole fitting procedure for rawacf data
 
     Calls the _fit procedure in a parallelized multiprocessing
     environment to speed up the procedure. The speed of this
     routine scales with number of cores.
 
     :param raw_records: a list of raw data dictionaries
+    :param       tdiff: Propagation time difference between arrays, in us.
     :returns: a list of dictionaries with fitted data
 
     """
-
     pool = mp.Pool()
-    fitted_records = pool.map(_fit,raw_records)
+    fitted_records = pool.map(partial(_fit, tdiff=tdiff), raw_records)
 
     return fitted_records
 
 
-
-
-if __name__ == "__main__":
+def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("infile", help="Input rawacf file path", type=str)
     parser.add_argument("outfile", help="Output file path", type=str)
+    parser.add_argument("--tdiff", help="Tdiff value to use for processing. If not specified, tdiff_a "
+                                        "from hdw file is used. Units of us", type=float, default=None)
     parser.add_argument("--debug-mode", help="""Disables multiprocessing usage to produce more
         meaningful exception at the cost of performance.""", action="store_true")
     args = parser.parse_args()
 
     raw_records = dm.parse_dmap_format_from_file(args.infile)
 
     if args.debug_mode:
-        fitted_records = [_fit(rr,True) for rr in raw_records]
+        fitted_records = [_fit(rr, args.tdiff, True) for rr in raw_records]
     else:
-        fitted_records = fit(raw_records)
+        fitted_records = fit(raw_records, args.tdiff)
 
     dm.dicts_to_file(fitted_records,args.outfile,file_type='fitacf')
+
+
+if __name__ == "__main__":
```

Comment:
I suppose we can search our usages of backscatter to see where we might be calling it like this.